2 - ListsLogicAndIterations/Code/Lists.cs

```diff
@@ -7,27 +7,39 @@ public static class Lists
 	{
 		public static List<decimal> BuildList(decimal first, decimal second)
 		{
-			throw new NotImplementedException();
+            var result = new List<decimal> { first, second };
+            return result;
 		}
 
 		public static decimal GetValue(List<decimal> list, int position)
 		{
-			throw new NotImplementedException();
+            try
+            {
+                return list[position];
+            }
+            catch
+            {
+                return 0;
+            }
+
 		}
 
 		public static void AddValue(List<decimal> list, decimal value)
 		{
-			throw new NotImplementedException();
+            list.Add(value);
 		}
 
 		public static void AddValue(List<decimal> list, List<decimal> value)
 		{
-			throw new NotImplementedException();
+			foreach(decimal number in value)
+            {
+                list.Add(number);
+            }
```

Comment:
This works, however there is a built in method that does this for you `list.AddRange(value);`